enterprise_access/apps/bffs/serializers.py

```diff
@@ -0,0 +1,40 @@
+"""
+Serializers for bffs.
+"""
+from rest_framework import serializers
+
+
+class ErrorSerializer(serializers.Serializer):
+    """
+    Serializer for error.
+
+    Fields:
+        user_message (str): A user-friendly error message.
+        developer_message (str): A more detailed error message for debugging purposes.
+    """
+    developer_message = serializers.CharField()
+    user_message = serializers.CharField()
+
```

Comment:
_Note_:  Something I considered but deferred on was to serialize the request url and params as part of exposing. pertinent metadata within the errors for better observability.